### ISSUE TITLE
I cannot install v2.9.1-alpha on 2 of my computers, splash screen disappears...

### ISSUE THREAD

**Issue by DG2YCB** [phw198/OutlookGoogleCalendarSync#1295]
> Checklist:-
> * [ ] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [ ] Reproduce the issue
> * [ ] Complete the information below
> * [ ] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v1.2.3.4  
**Edition**: v2.9.1-alpha Installed  
**Sync Direction**: all

**Bug Description**  
I cannot install v2.9.1-alpha on 2 of my computers, because after each start splash screen comes and browser is opened for displaying release notes. but then nothing more happens. Splash screen disappears but program is not started. Occured on Windows 7 as well as on 1 of my Windows 10 computers.

**Steps to Reproduce the Issue**  
1. Download the installer and try to install it
2. On the desktop an icon appears, but when clicking on that only the splash screen comes and a browser is launched. Then the splash screen disappears, but nothing more happens.

**Any other information**  
Any additional information that may help troubleshoot the issue.  
:paperclip: Attach your `OGcalsync.log` file.

**Comment by phw198** [comment]
:spiral_notepad: Please provide your `OGcalsync.log` file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).

**Comment by DG2YCB** [comment]
Please find attached the initial log file from my notebook (when splash screen disappeared but nothing more happened). Meanwhile I brought it to work on all my computers. Just copied the entire OGCS entries under AppData from my computer where installation was successful to all other computers. I needed to run through Google authentification once again but at least the OGCS program started. 

One issue I have is that each time I start a computer, OGCS fails to sync because of Google authentification failure. I have to run through Google authentification process once again. Frankly, if that happens every day, then ODCS turns to be not very attractive. Is this normal behaviour or might there be something wrong with my installations? Or is it not possible to run OGCS on more than one computer?

Regards, Uwe

[OGcalsync.log.2021-09-23_from_my_notebook.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/7227191/OGcalsync.log.2021-09-23_from_my_notebook.txt)

**Comment by phw198** [comment]
The error message you're receiving from Outlook is
```
System.Runtime.InteropServices.COMException: Dieser Vorgang konnte wegen eines Registrierungs- oder Installationsproblems nicht ausgeführt werden. Starten Sie Outlook neu, und versuchen Sie es erneut. Falls das Problem weiterhin besteht, installieren Sie Outlook erneut.
Code: 0x9CD4011C,-1663827684; Location: get_ExchangeMailboxServerVersion() in <Unknown File>
```
Which translates to:

> This operation could not be performed because of a registration or installation problem. Restart Outlook and try again. If the problem persists, reinstall Outlook.

**Comment by phw198** [comment]
:package: The error message should be displayed before OGCS closes with [hotfix v2.9.1.3.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/7239418/v2.9.1.3.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).
